src/routes/index.ts

```diff
@@ -1,8 +1,10 @@
 import express from 'express';
+import { deviceController } from '../controllers/device.controller';
 import { healthController, notificationController } from '../controllers';
```

Comment:
You are exporting `deviceController` from `src/controllers/index.ts`, so you do not need a separate import.

Instead, you can add it to the bulk import, and since that makes the line so long, we can spill it to multiple lines:

```suggestion
import {
  deviceController,
  healthController,
  notificationController,
} from '../controllers';
```

Reply:
I thought I clubbed them together.. Anyway I will change it.